projects/storefrontlib/shared/components/modal/modal.service.ts

```diff
@@ -39,21 +39,21 @@ export class ModalService {
     });
   }
 
-  getActiveModal(): ModalRef {
+  getActiveModal(): ModalRef | null {
     const modal = this.modals[this.modals.length - 1];
     return modal ? modal : null;
   }
 
   dismissActiveModal(reason?: any): void {
-    const modal: ModalRef = this.getActiveModal();
+    const modal: ModalRef | null = this.getActiveModal();
```

Comment:
I think we can just remove : ModalRef | null since it's implicit from the method

Reply:
same for the bottom one